src/main/java/Spike.java

```diff
@@ -0,0 +1,135 @@
+            }
+            iter += 1;
+        }
+        displayByeMsg();
+    }
+
+    private static void displayAcknowledgement(Task[] inputList, int iter) {
+        char Badge = getBadge(inputList[iter]);
+        System.out.println(DIVIDER);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(" [" + Badge + "]" + "[ ] " + inputList[iter].description);
+        System.out.println("Now you have " + (iter + 1) + " tasks in the list.");
+        System.out.println(DIVIDER);
+    }
+
+    private static String processEvent(String input) {
+        String event = input.substring(6);
+        String[] parts = event.split(" /from ");
+        String[] time = parts[1].split(" /to ");
+        return parts[0] + " (from: " + time[0] + " to: " + time[1] + ")";
+    }
+
+    private static String processDeadline(String input) {
+        String deadline = input.substring(9);
+        String[] parts = deadline.split(" /by ");
+        return parts[0] + " (by: " + parts[1] + ")";
+    }
+
+    private static String processTodo(String input) {
+        return input.substring(5);
+    }
+
+    private static void displayAddedMsg(String input) {
+        System.out.println(DIVIDER);
+        System.out.println("added: " + input);
+        System.out.println(DIVIDER);
+    }
+
+    private static void displayList(Task[] inputList) {
+        System.out.println(DIVIDER);
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < inputList.length; i++) {
+            Task value = inputList[i];
+            if (value == null) {
+                break;
+            }
+            char Badge = getBadge(value);
+            System.out.println((i + 1) + ".[" + Badge + "]" + "[" + value.getStatusIcon() + "] " + value.description);
+        }
+        System.out.println(DIVIDER);
+    }
+
+    private static char getBadge(Task value) {
+        char Badge = ' ';
+        if (value instanceof Todo) {
+            Badge = ((Todo) value).getBadge();
+        } else if (value instanceof Deadline) {
+            Badge = ((Deadline) value).getBadge();
+        } else if (value instanceof Event) {
+            Badge = ((Event) value).getBadge();
+        }
+        return Badge;
+    }
+
+    private static void displayUnmarkMsg(int index, Task[] inputList) {
+        char Badge = getBadge(inputList[index]);
+        System.out.println(DIVIDER);
+        System.out.println("OK, I've marked this task as not done yet:");
+        System.out.println((index + 1) + ".[" + Badge + "]" + "[ ] " + inputList[index].description);
+        System.out.println(DIVIDER);
+    }
+
+    private static void displayMarkMsg(int index, Task[] inputList) {
+        char Badge = getBadge(inputList[index]);
+        System.out.println(DIVIDER);
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println((index + 1) + ".[" + Badge + "]" + "[X] " + inputList[index].description);
```

Comment:
Avoid long lines of code
Try to add line breaks to reduce the length of code on each line (<110 - 120 characters, ideally 80 characters)